components/card/AddDetails.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { useCardStore } from '~/store/cardStore';
+import type { Type } from '~/types/models/Card';
+import { computed } from 'vue';
+
+const newCardStore = useCardStore();
+
+const props = defineProps<{
+    fieldTypes: Type[];
+}>();
+
+const { personalFields } = storeToRefs(newCardStore);
+
+
+const updateTitle = (title: string, category: string) => {
+    newCardStore.setCurrentCategory(category);
+    newCardStore.setEditing(false);
+    newCardStore.setModalTitle(title);
+    newCardStore.setModalOpen(true);
+};
+
+const isFieldDisabled = (fieldName: string) => {
+    switch (fieldName) {
+        case 'Name':
+            return personalFields.value.find((field: any) => field?.name === 'Name') !== undefined;
+        case 'Job Title':
+            return personalFields.value.find((field: any) => field?.name === 'Job Title') !== undefined;
+        case 'Department':
+            return personalFields.value.find((field: any) => field?.name === 'Department') !== undefined;
+        case 'Company Name':
+            return personalFields.value.find((field: any) => field?.name === 'Company Name') !== undefined;
+        default:
+            return false;
+    }
+};
+
```

Comment:
why we still have this ?

Reply:
I will be revised once matapos na po yung dashboard revision po dahil components yan ng create/edit